docs/app/interfaces/IPlunk.ts

```diff
@@ -1,3 +1,13 @@
 export interface IPlunk {
-    [key: string]: string;
-}
+    files: {
+        [key: string]: string;
+    };
+    modules?: {
+        imports?: string[];
+        library?: string;
+        map?: {
```

Comment:
We should probably make this an array as we may want to provide multiple mappings